src/support/restore-securities.ts

```diff
@@ -1,4 +1,9 @@
 import * as _ from 'lodash'
+import * as Bluebird from 'bluebird'
+
+// I can't find correct Bluebird types, I tried the package but he doesn't knows
+// Bluebird.map
+const OiseauBleu: any = Bluebird
```

Comment:
WAT
The `@types/bluebird` doesn't work?

Reply:
BTW we should try to code an entire application with full french naming.

Reply:
I've just tried it on sandbox it worked for me as expected.
If you remove `:any` type and hover the OiseauBlue variable does it say  

![image](https://user-images.githubusercontent.com/10114634/85110482-45990380-b213-11ea-9cc4-c263d00c92c0.png)

Reply:
@rolljee Have you tried to call the `Bluebird.map` method?